packages/grafana-data/src/transformations/fieldReducer.ts

```diff
@@ -305,6 +405,23 @@ export const fieldReducers = new Registry<FieldReducerInfo>(() => [
   },
 ]);
 
+for (let i = 1; i < 100; i++) {
+  const percentile = i / 100;
+  const id = `p${i}` as ReducerID;
+  const nth = (n: number) => n>3&&n<21?"th":n%10===1?"st":n%10===2?"nd":n%10===3?"rd":"th";
```

Comment:
I like this.

Reply:
You can thank the internet for that one 😂